glide.yaml

```diff
@@ -0,0 +1,69 @@
+package: github.com/openshift/origin
+excludeDirs:
+- cmd/cluster-capacity
+- cmd/service-catalog
+- pkg/build/builder
+import:
+- package: github.com/coreos/etcd
+  version: v3.2.8
+- package: github.com/gonum/graph
+  version: bde6d0fbd9dec5a997e906611fe0364001364c41
+- package: github.com/google/certificate-transparency
+  repo:    git@github.com:openshift/google-certificate-transparency
+  version: master
+- package: github.com/vishvananda/netlink
+  version: 933b978eae8c18daa1077a0eb7186b689cd9f82d
+- package: github.com/google/cadvisor
+  repo:    git@github.com:openshift/google-cadvisor
+  version: release-0.27.1
+- package: github.com/emicklei/go-restful-swagger12
+  repo:    https://github.com/openshift/emicklei-go-restful-swagger12
+  version: release-1.0.1
+- package: github.com/docker/docker
+  version: 4f3616fb1c112e206b88cb7a9922bf49067a7756
+- package: github.com/fsouza/go-dockerclient
+  version: 3f9370a4738ba8d0ed6eea63582ca6db5eb48032
+- package: github.com/cloudflare/cfssl
+  repo:    https://github.com/openshift/cloudflare-cfssl
+  version: stable-20160905
+- package: github.com/skynetservices/skydns
+  repo:    https://github.com/openshift/skynetservices-skydns
+  version: release-2.5.3a
+- package: k8s.io/gengo
+  repo:    git@github.com:openshift/kubernetes-gengo
+  version: openshift-3.8
+- package: k8s.io/code-generator
+  repo:    git@github.com:openshift/kubernetes-code-generator
+  version: release-1.8.1
+- package: k8s.io/apimachinery
+  repo:    git@github.com:openshift/kubernetes-apimachinery
+  version: release-1.8.1
+- package: k8s.io/api
+  repo:    git@github.com:openshift/kubernetes-api
+  version: release-1.8.1
+- package: k8s.io/client-go
+  repo:    git@github.com:openshift/kubernetes-client-go
+  version: release-1.8.1
+- package: k8s.io/metrics
+  repo:    git@github.com:openshift/kubernetes-metrics
+  version: release-1.8.1
+- package: k8s.io/apiserver
+  repo:    git@github.com:openshift/kubernetes-apiserver
+  version: release-1.8.1
+- package: k8s.io/kube-aggregator
+  repo:    git@github.com:openshift/kube-aggregator
+  version: release-1.8.1
+- package: k8s.io/apiextensions-apiserver
+  repo:    git@github.com:openshift/kubernetes-apiextensions-apiserver
+  version: release-1.8.1
+- package: k8s.io/kubernetes
+  repo:    git@github.com:openshift/kubernetes
+  version: release-1.8.1
+# only used by pkg/build/builder
+- package: github.com/kubernetes-incubator/cri-o
+  version: a8ee86b1cce0c13bd541a99140682a92635ba9f7
+- package: github.com/containers/image
+  repo:    git@github.com:openshift/containers-image
+  version: openshift-3.8
```

Comment:
so does this mean that if I want to bump this I will have to bump the fork branch and then re-run update-deps ?

Reply:
> so does this mean that if I want to bump this I will have to bump the fork branch and then re-run update-deps ?

Yeah. Or you can find the person who made that ridiculous carry and make them fix it.